init_coupler.cfg

```diff
@@ -60,7 +60,6 @@ dt_initial                  = 4e2    # Inital step size
 # Flux convergence scheme and tolerances for surface equilibration
 flux_convergence            = 0         # 0: off | 1: on | 2: restart
 F_atm_bc                    = 0         # Boundary condition choice for F_atm, 0: TOA | 1: Surface
-F_crit                      = 0.1       # Critical flux, below which attempt to stabilise the evolution (0: disabled)
```

Comment:
Why is this gone?

Reply:
It wasn't being used by the version of PROTEUS that I branched for these changes. However, I've now repurposed this flag further down in the cfg file as a check for when the planet intends to heat up, since we assume secular cooling. I've tried to explain this a bit in the documentation updates.

Reply:
It wasn't being used in any of the default cases or the grid. However, I repurposed this configuration option further down the cfg file as a check for when the planet intends to heat up, since we assume secular cooling.